torch/_dynamo/eval_frame.py

```diff
@@ -590,14 +590,15 @@ def result_capturing_wrapper(*graph_inputs):
 
     remove_from_cache(f)
     with patch(f"{__name__}.most_recent_backend", None):
-        opt_f = optimize_assert(
-            dynamo_normalization_capturing_compiler,
-            hooks=Hooks(guard_export_fn=guard_export_print, guard_fail_fn=None),
-            export=True,
-            dynamic=(tracing_mode == "symbolic"),
-        )(f)
-        # TODO(voz): We may have instances of `f` that mutate inputs, we should track sideffects and reject.
-        result_traced = opt_f(*args, **kwargs)
+        with config.patch(specialize_int_float=True):
```

Comment:
you can put the two context managers together: `with patch(...), config.patch(...):`